### PR TITLE
Add updatePrice and checkPriceStaleness tasks

### PR DESCRIPTION
**Pull request by jerryfan01234**
Apologies for the large PR. Normally I would've broken into smaller PRs but time zone difference.

Main changes:
1. Implemented updatePrice task with real chain calls and calculations. The tests have not successfully run due to missing insufficient ETH funds. After ETH funds transferred, will run tests and submit follow up PR if fixes needed.

2. Added checkPriceStaleness task. Did not have time to write unit tests. 

3. Some refactoring

### CODE REVIEW

**Review by zmckinnon — APPROVED** [review]

File: task-looper/src/__tests__/utils/math.test.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    });
+
+    it('should handle multiplication with a negative multiplier', () => {
+        const result: bigint = bigintNumberMultiplication(BigInt(100), -3.5);
+        expect(result).toBe(BigInt(-350));
+    });
+
+    it('should handle multiplication with a very small multiplier', () => {
+        const result: bigint = bigintNumberMultiplication(BigInt(10), 0.000000001);
+        expect(result).toBe(BigInt(0));
```
Comment (eyqs) [inline task-looper/src/__tests__/utils/math.test.ts:31]: You mean, it should round to the nearest integer after multiplication with a very small multiplier

File: task-looper/src/tasks/updatePrice.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    // Wait for block to commit
+    await publicClient.waitForTransactionReceipt({ hash });
+
+    // Verify price updated correctly in chain
+    const chainAskPrice: bigint = await getAskPrice(runConfig.address as `0x${string}`, runConfig.abi);
+    if (chainAskPrice != newAskPrice) {
+        // TODO: publish metric `updatePrice mismatch` with alarms on metric
+        throw new Error(`askPrice not updated in chain, attempted to set askPrice to ${newAskPrice} but chain value is ${chainAskPrice} `)
+    }
```
Comment (eyqs) [inline task-looper/src/tasks/updatePrice.ts:37]: Can you set the bid price as well?

File: task-looper/src/config.ts [hunk 3]
```diff
@@ -1,10 +1,34 @@
...
+import { Abi } from "viem";
+import { checkPriceStaleness } from "./tasks/checkPriceStaleness";
+
+const updatePriceRunConfigs = [
+    {
+        name: "NEV",
+        cadence: 10,
+        address: "0x659619AEdf381c3739B0375082C2d61eC1fD8835",
+        abi: NEVABIJson as Abi,
+        r: 0.1,
```
Comment (eyqs) [inline task-looper/src/config.ts:12]: Consider using `decimal.js` to handle decimal calculations

**Review by eyqs — APPROVED** [review]
LGTM

File: task-looper/src/config.ts (hunk 3 above)
Comment (jerryfan01234) [reply task-looper/src/config.ts:12]: gotcha, will leave as a TODO item

**Pull request merged**